### PR TITLE
Fix bug in type validation.

### PR DESCRIPTION
**Pull request by jmccrosky**
I think I found a bug, but please confirm that I'm not messing anything up here.

### CODE REVIEW

File: simpleprophet/simpleprophet/data.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def get_kpi_data(bq_client, types=tuple(KPI_QUERIES.keys())):
             if q == k.lower():
                 q = k
                 break
-        else:
+        if q not in KPI_QUERIES.keys():
```
Comment (jklukas) [inline simpleprophet/simpleprophet/data.py:63]: The `else` here is intentional. It should be executed only if the `for` loop runs to completion without any of the cases `break`ing out. It's a lesser-used feature, so I can add a comment here to clarify.

Have this caused any functional problems?

Comment (jmccrosky) [reply simpleprophet/simpleprophet/data.py:63]: I think it means that the types argument has to be lower case or we raise an error?  Or am I misunderstanding?

Comment (jklukas) [reply simpleprophet/simpleprophet/data.py:63]: The intention was to accept any casing for the input and this code would normalize the value of `q` to match the expected casing. But I think you're correct that this now only supports `q` being lowercase. Will fix.

Comment (jklukas) [reply simpleprophet/simpleprophet/data.py:63]: Actually, I like your solution. Let's merge it.

**Review by jklukas — APPROVED** [review]

**Pull request merged**